.bumpversion.cfg

```diff
@@ -1,20 +1,12 @@
 [bumpversion]
 commit = False
 tag = False
-current_version = 2.0.0
+current_version = 2.1.0
 
 [bumpversion:file:README.rst]
 
-# [bumpversion:file:README.EN.rst]
-
 [bumpversion:file:package.json]
 search = "version": "{current_version}",
 replace = "version": "{new_version}",
 
 [bumpversion:file:src/openklant/__init__.py]
-
-# [bumpversion:file:docs/api/index.rst]
-
-# [bumpversion:file:publiccode.yaml]
-# search = softwareVersion: {current_version}
-# replace = softwareVersion: {new_version}
```

Comment:
These were removed automatically. IDK if that matters

Reply:
Sure let's remove commented code